src/main/java/ch/njol/skript/SkriptEventHandler.java

```diff
@@ -119,60 +123,60 @@ private static void check(Event e, EventPriority priority) {
 				return;
 			Class<? extends Event> c = e.getClass();
 			ts = getTriggers(c);
-			
+
 			logEventStart(e);
 		}
-		
+
 		if (e instanceof Cancellable && ((Cancellable) e).isCancelled() && !listenCancelled.contains(e.getClass()) &&
-				!(e instanceof PlayerInteractEvent && (((PlayerInteractEvent) e).getAction() == Action.LEFT_CLICK_AIR || ((PlayerInteractEvent) e).getAction() == Action.RIGHT_CLICK_AIR) && ((PlayerInteractEvent) e).useItemInHand() != Result.DENY)
-				|| e instanceof ServerCommandEvent && (((ServerCommandEvent) e).getCommand().isEmpty() || ((ServerCommandEvent) e).isCancelled())) {
+			!(e instanceof PlayerInteractEvent && (((PlayerInteractEvent) e).getAction() == Action.LEFT_CLICK_AIR || ((PlayerInteractEvent) e).getAction() == Action.RIGHT_CLICK_AIR) && ((PlayerInteractEvent) e).useItemInHand() != Result.DENY)
+			|| e instanceof ServerCommandEvent && (((ServerCommandEvent) e).getCommand().isEmpty() || ((ServerCommandEvent) e).isCancelled())) {
```

Comment:
This condition is not eye friendly, it's better to make a local variable for each condition part and then use those in one `if` block
e.g.
```java
boolean isBla = x || y;
boolean isBoo = a && b;
boolean isBlo = c && d;
if (isBla || isBoo && isBlo)
```